js/dialog.ts

```diff
@@ -181,6 +186,38 @@ export class Dialog {
         this.SetDefaultProperty(DialogElement.TITLE, 'background', '#9d2c2c')
         this.SetDefaultProperty(DialogElement.TITLE, 'color', 'whitesmoke')
     }
+    private MultiFileDialog() {
+        let row = document.createElement('div')
+        let input = document.createElement('input')
+        let button = document.createElement('button')
+
+        row.classList.add('dialog-element', 'flex-row', 'file-input-control')
+        input.type = 'file'
+        input.multiple = true;
+        button.innerHTML = 'Select Files'
+        button.onclick = function (event) {
+            event.preventDefault();
+            input.click();
+        }
+
+        let _this = this
+        input.onchange = function (event) {
+            if (!event.target) return
+            _this.data.file_list = (<HTMLInputElement>event.target)?.files
+            if (_this.data.file_list) {
+                _this.accept()
```

Comment:
I'd prefer it if, similarly to how the ‘Select an image’ dialog works, selected images were first displayed in the dialog (their filenames that is), and only if the user then pressed ‘Ok’ would they then actually get added as characters.